.github/workflows/specs-model-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Specs Model - Test
+
+on:
+  push:
+    branches:
+      - main
+      - typespec-next
+      - users/kojamroz/specs_model_1
+  pull_request:
+    paths:
+      - package-lock.json
+      - package.json
+      - tsconfig.json
+      - .github/workflows/_reusable-eng-tools-test.yaml
+      - .github/workflows/specs-model-test.yaml
+      - eng/tools/package.json
+      - eng/tools/tsconfig.json
+      - eng/tools/specs-model/**
+
+jobs:
+  specsModel:
+    name: Specs Model
+    uses: ./.github/workflows/_reusable-eng-tools-test.yaml
+    with:
+      package: specs-model
+      lint: true
```

Comment:
FYI while I added mandatory eslint and prettier checks to specs-model, I would be in favor in enabling it for everything in `eng`. I am happy to help adjusting the existing code and linting rules to make eslint and prettier pass on everything.